core/src/main/java/org/testcontainers/containers/output/BaseConsumer.java

```diff
@@ -0,0 +1,76 @@
+package org.testcontainers.containers.output;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+
+@SuppressWarnings("Duplicates")
```

Comment:
I wonder what does it duplicate?

Reply:
Argh... It's just a refactoring artifact. Will remove this.